sources/packages/backend/apps/queue-consumers/src/processors/schedulers/esdc-integration/ecert-integration/ecert-full-time-process-integration.scheduler.ts

```diff
@@ -0,0 +1,50 @@
+import { InjectQueue, Process, Processor } from "@nestjs/bull";
+import { ECertFileHandler } from "@sims/integrations/esdc-integration";
+import { QueueService } from "@sims/services/queue";
+import { QueueNames } from "@sims/utilities";
+import { Job, Queue } from "bull";
+import { QueueProcessSummary } from "../../../models/processors.models";
+import { BaseScheduler } from "../../base-scheduler";
+import { ESDCFileResult } from "../models/esdc";
+
+@Processor(QueueNames.FullTimeECertIntegration)
+export class FullTimeECertProcessIntegrationScheduler extends BaseScheduler<void> {
+  constructor(
+    @InjectQueue(QueueNames.FullTimeECertIntegration)
+    schedulerQueue: Queue<void>,
+    queueService: QueueService,
+    private readonly eCertFileHandler: ECertFileHandler,
+  ) {
+    super(schedulerQueue, queueService);
+  }
+
+  /**
+   * Process Full-Time disbursements available to be sent to ESDC.
+   * Consider any record that is scheduled in upcoming days or in the past.
+   * @params job job details.
+   * @returns result of the file upload with the file generated and the
+   * amount of records added to the file.
+   */
+  @Process()
+  async processFullTimeECert(job: Job<void>): Promise<ESDCFileResult> {
+    const summary = new QueueProcessSummary({
+      appLogger: this.logger,
+      jobLogger: job,
+    });
+    await summary.info(
+      `Processing E-Cert fulltime integration job ${job.id} of type ${job.name}.`,
+    );
+    await summary.info("Sending Full-Time E-Cert File...");
```

Comment:
Please try to be consistent while referring to Full-time/Part-time.
In the wiki, we can see "Full-time/Part-time" and "Full time/Part time".
I would be giving this comment if they were consistent at least in the same file.
Line 37 can be changed as below to be in sync with line 35
Or both can be changed to be in sync with the comment at line 22.